### PR TITLE
Improve LabelModel Performance

### PR DESCRIPTION
**Pull request by paroma**
* Change seed for LabelModel
* Change accuracy to F1

### CODE REVIEW

File: spouse/spouse_demo.py [hunk 1]
```diff
@@ -414,7 +414,13 @@ def lf_distant_supervision_last_names(x, last_names):
 from snorkel.labeling.model.label_model import LabelModel
 
 label_model = LabelModel(cardinality=2, verbose=True)
-label_model.fit(train_L, dev_labels, n_epochs=5000, log_freq=500, seed=123)
+label_model.fit(train_L, dev_labels, n_epochs=500, log_freq=50, seed=12345)
+
+from snorkel.analysis.metrics import metric_score
```
Comment (brahmaneya) [inline spouse/spouse_demo.py]: I think these four lines are already in the next cell
